mlflow/tracking/client.py

```diff
@@ -197,19 +198,6 @@ def set_terminated(self, run_id, status=None, end_time=None):
                                    end_time=end_time)
 
 
-def get_service(tracking_uri=None):
```

Comment:
There's still a reference to `get_service` at `docs/source/tracking.rst`.

Reply:
Good catch! Thanks...